draft-trammell-taps-post-sockets.md

```diff
@@ -435,14 +436,24 @@ reuse an Association if they can be carried from the same Local to the same
 Remote over the same Paths; this re-use of an Association may implies the
 creation of a new Transient.
 
+Associations may exist without a Message Carrier if required. This may be done if
+peer cryptographic state, e.g., a pre-shared key, is established out-of-band.
+Thus, Associations may be created without the need to send application data
+to a peer, i.e., without a Remote. Associations may also be created from
```

Comment:
When does an implementation choose to create an association from another one, rather than adding the keys to an existing one? Any given association represents state with one remote, as far as my impression goes.